extras/nginx/nginx-1.16.patch

```diff
@@ -3923,6 +3942,16 @@ index 00000000..5bbff860
 +        return ngx_http_next_header_filter(r);
 +    }
 +
++    if (r->header_sent) {
++        return NGX_OK;
++    }
++
++    r->header_sent = 1;
```

Comment:
This can't be set before we actually send the headers, as it would break sending of responses that are blocked. It is already set inside `ngx_http_v3_send_response()` so there is no need to set it again here.

Reply:
Hi, I described in more detail (about r->header_sent) in the new pr https://github.com/cloudflare/quiche/pull/504 (https://github.com/cloudflare/quiche/pull/504/commits/938e9154dae0b0e761913f7f4fd5f7569dc68ae5)